src/main/scala/nl.knaw.dans.easy.bag2deposit/package.scala

```diff
@@ -57,11 +58,23 @@ package object bag2deposit extends DebugEnhancedLogging {
 
   def loadXml(file: File): Try[Elem] = {
     trace(file)
-    Try(XML.loadFile(file.toJava))
-      .recoverWith {
-        case t: FileNotFoundException => Failure(InvalidBagException(s"could not find: $file"))
-        case t: SAXParseException => Failure(InvalidBagException(s"could not load: $file - ${ t.getMessage }"))
-      }
+    Try {
+      val withoutPrologue = file.contentAsString
+        .replaceAll("<[?].+[?]>", "")
+      val dd = "<([0-9a-fA-F]{2})>"// reuse within brackets is a problem
+      val string = // make sure to have a prologue
+        """<?xml version="1.0" encoding="UTF-8" ?>
+          |""".stripMargin + withoutPrologue
+          // preparations for UnicodeRewriteRule
+          .replaceAll(s"(?s)<([cCdD][0-9a-fA-F])>$dd","<hack:encoded>$1$2</hack:encoded>")
+          .replaceAll(s"(?s)<([eE][0-9a-fA-F])>$dd$dd","<hack:encoded>$1$2$3</hack:encoded>")
+          .replaceAll(s"(?s)<([fF][0-9a-fA-F])>$dd$dd$dd","<hack:encoded>$1$2$3$4</hack:encoded>")
```

Comment:
The last example of https://www.scala-lang.org/api/2.12.1/scala/util/matching/Regex.html looks promising to eliminate `UnicodeRewriteRule`.